theme/snippets/add-to-cart-container.liquid

```diff
@@ -27,6 +27,11 @@
     {% assign ends_at = bookings_metafields.ends_at | date: '%s' %}
   {% endif %}
 
+  {% if product.metafields.accentuate.starts_at %}
```

Comment:
What if `starts_at` is present, but `ends_at` is not?

Reply:
Good catch. I changed this to check for both to exist for the theme to use the values from `accentuate`. I think this makes sense because if an editor only adds one and forgets the other, but the theme uses one, the editor might think it worked. 

But now, if an editor forgets one, they would see the theme not using either and hopefully go back and fix it.